### PR TITLE
Content check and confirm change of cycle

### PR DESCRIPTION
**Pull request by laurahermionetennant92**
Made some content changes to the explanation of what providers are doing when they change a cycle. 
Along the way, realised I don't totally understand the need for a radio button here as keeping the offer in the current cycle is the current situation (so not another option for them to choose from if that makes sense):
Screenshot 2020-07-08 at 12 47 44

### CODE REVIEW

File: app/views/_includes/application/details.njk [hunk 1]
```diff
@@ -17,7 +17,7 @@
       items: [
         {
           href: '/application/' + application.id + '/cycle/edit',
-          text: "Change",
+          text: "Defer offer",
```
Comment (adamsilver) [inline app/views/_includes/application/details.njk:20]: We considered this but it is inconsistent with all the other change links we use and it also means the link text would have to change to ‘Undefer’ (or similar).

Users have the burden of rediscovering it.

That said I think we need to keep an eye on research to grasp people's mental model.

File: app/views/application/cycle/edit/check.njk [hunk 2]
```diff
@@ -19,18 +19,20 @@
         {{name}}
       </span>
       <h1 class="govuk-heading-xl">
-        Check and confirm change of cycle
+        Check and confirm deferral to next cycle
       </h1>
 
-      <p>The course you have currently offered is not available yet in the next cycle.</p>
+      <p>You are deferring this offer to the next cycle (2020 to 2021).</p>
```
Comment (adamsilver) [inline app/views/application/cycle/edit/check.njk:25]: Aren't we telling them exactly what they already know here?

File: app/views/application/cycle/edit/cycle.njk [hunk 3]
```diff
@@ -20,20 +20,20 @@
           name: "applicatoncycle",
           fieldset: {
             legend: {
-              text: "Change cycle",
+              text: "Defer offer to next cycle",
```
Comment (adamsilver) [inline app/views/application/cycle/edit/cycle.njk:23]: This no longer works when switching back to the previous cycle.

File: app/views/application/cycle/edit/check.njk (hunk 2 above)
Comment (adamsilver) [reply app/views/application/cycle/edit/check.njk:25]: The most crucial part of the guidance now has been removed.

File: app/views/_includes/application/details.njk (hunk 1 above)
Comment (laurahermionetennant92) [reply app/views/_includes/application/details.njk:20]: Ah OK

File: app/views/application/cycle/edit/check.njk (hunk 2 above)
Comment (laurahermionetennant92) [reply app/views/application/cycle/edit/check.njk:25]: No, it's the second sentence :)

File: app/views/application/cycle/edit/cycle.njk (hunk 3 above)
Comment (laurahermionetennant92) [reply app/views/application/cycle/edit/cycle.njk:23]: OK

Comment (laurahermionetennant92) [reply app/views/application/cycle/edit/cycle.njk:23]: Are you saying that on this page, you might use the green button to revert back to the old cycle? Sorry don't understand
Screenshot 2020-07-08 at 14 49 01

File: app/views/application/cycle/edit/check.njk (hunk 2 above)
Comment (adamsilver) [reply app/views/application/cycle/edit/check.njk:25]: Oh fab.

Comment (adamsilver) [reply app/views/application/cycle/edit/check.njk:25]: Still think the first line is redundant

File: app/views/application/cycle/edit/check.njk [hunk 4]
```diff
@@ -19,18 +19,20 @@
...
 
-      <p>This application will be marked as deferred. You must reinstate your offer at the start of the next cycle.</p>
+      <p>The course you are offering is not open for that year yet. When the next cycle starts, we’ll prompt you to check and confirm the details of your offer.</p>
 
-      <p>The candidate will be sent a confirmation by email of this action after you confirm the change of cycle.</p>
+      <p>You'll be able to change the course, location or provider if you need to.</p>
```
Comment (adamsilver) [inline app/views/application/cycle/edit/check.njk:29]: If calling out things then mention conditions?

**Comment by laurahermionetennant92** [comment]
@adamsilver Have made those tweaks we discussed :)

**Pull request closed without merging**